src/quotamodel.c

```diff
@@ -785,13 +787,13 @@ refresh_disk_quota_model(bool is_init)
  * process is constructing quota model.
  */
 static void
-refresh_disk_quota_usage(bool is_init)
+refresh_disk_quota_usage(DiskquotaDBEntry *dbEntry)
 {
 	bool  connected                   = false;
 	bool  pushed_active_snap          = false;
 	bool  ret                         = true;
+	bool  is_init                     = !dbEntry->inited;
 	HTAB *local_active_table_stat_map = NULL;
-
```

Comment:
Do not remove empty line between arguments and code.

Reply:
Done